.github/workflows/ci-aarchxx.yml

```diff
@@ -75,6 +75,28 @@ jobs:
       env:
         DYNAMORIO_CROSS_AARCHXX_LINUX_ONLY: yes
 
+    - name: Send failure mail to dynamorio-devs
+      if: failure()
+      uses: dawidd6/action-send-mail@v2
+      with:
+        server_address: smtp.gmail.com
+        server_port: 465
+        username: ${{secrets.NOTIFICATION_EMAIL_USERNAME}}
+        password: ${{secrets.NOTIFICATION_EMAIL_PASSWORD}}
+        subject: Github Actions CI FAILED on ${{github.repository}} ${{github.ref}}
```

Comment:
I would say include `github.event_name` so we know whether it's `pull request` or whatever the string is for a cronjob at a glance.

Maybe include github.workflow in the title too so we can tell at a glance whether it's osx or windows or what?

Reply:
Done.